2-repositories/iglu-server/src/test/scala/com/snowplowanalytics/iglu/server/service/SchemaServiceSpec.scala

```diff
@@ -33,10 +33,11 @@ class SchemaServiceSpec extends org.specs2.Specification { def is = s2"""
     Returns public and private schemas with apikey $e8
     Return error for non-UUID apikey $e6
   PUT
-    Prohibits adding new schema if it already exists
-    Prohibits adding new schema if previous version does not exist
+    Prohibits adding new schema if it already exists $e11
+    Prohibits adding new schema if previous version does not exist $e12
     Prohibits adding new schema if previous version belongs to different apikey
```

Comment:
Is this the intended behavior? Updating a schema created by a (different) master key seems to work fine, and at a glance there's nothing in the logic that checks for api keys.

Reply:
I think by "different" I meant a key with unrelated vendor.